sto/ethereum/utils.py

```diff
@@ -347,18 +350,21 @@ def get_contract_deployed_tx(dbsession, contract_name):
     ).first()
 
 
-def whitelist_kyc_address(config, address, kyc_contract_address):
+def whitelist_kyc_address(config, address, kyc_contract_address,
+                          do_broadcast=True,
```

Comment:
Although `do_broadcast` is set here to true to reflect previous functionality, lets it set to false by default.
The rationale behind this is that we want to change the pattern used in this code. It shouldn't broadcast by itself.

Reply:
Changed. Ty!